project_PassMeCash_Polkadot.md

```diff
@@ -0,0 +1,165 @@
+# PassMeCash
+
+- **Team Name:** CodeUp Technologies Limited
+- **Payment Address:** 0x65d68c8d14cc4acf02ec4e8d572399620a684b57 (ERC 20)
+- **[Level]:** 2
+
+
+### Overview
+
+PassMeCash is a decentralized banking system that meet the financial needs of the public through the public using crypto (DOT) & fiat. PassMeCash is a web3 financial solution that will help our target audience access their funds for payment transactions seamlessly and effortlessly through our decentralized agents.
+Unlike the traditional centralized banking system where accounts holder funds are kept in bank building and also withdrawn from same bank building; PassMeCash is a community and decentralized financial solution, where the PassMeCash users can request for their funds and it will be available to them for withdrawal via the nearest PassMeCash agent to them. 
+PassMeCash is revolutionizing the banking and financial system by bringing funds to its users upon their request, instead of them going to banks to get funds. Users or Businesses can send and receive payments for goods and services via PassMeCash. Cross border payments and transfer can also be done through PassMeCash. Users who need low fiat currency bills to spray celebrants in parties and social functions, can also request for low currency bills through PassMeCash. All transactions on PassMeCash can be done using crypto (DOT) or cash payments.
```

Comment:
1. Yes, low currency bills simply mean low denominational currency, just like in the USA, the low currency bill is USD 1. But in the case of Nigeria we have NGN 20, NGN 50 & NGN 100 as low  currency bills.
2. In Nigeria, most people buy low currency bills which they throw around in parties or clubs, mostly to the celebrants of these parties. Thus through the PassMeCash Platform you can order for these low bills via your PassMeCash account either in crypto or fiat.
3. Yes, we will implement stablecoins like USDC or USDT, because stablecoin will give us the USD value which will be swapped to the Nigerian Naira (NGN). This swapping will occur within split seconds, without the user knowing, but at the end of transaction, the user can make cash payment using crypto. The process will go thus DOT-USDC to USD-NGN, or DOT-USDT to USD-NGN, and vice versa.